packages/transactions/README.md

```diff
@@ -51,7 +51,6 @@ const txOptions = {
   senderKey: 'b244296d5907de9864c0b0d51f98a13c52890be0404e83f273144cd5b9960eed01',
   network,
   memo: 'test memo',
-  nonce: new BigNum(0), // set a nonce manually if you don't want builder to fetch from a Stacks node
   fee: new BigNum(200), // set a tx fee if you don't want the builder to estimate
```

Comment:
`bn.js` instances are no longer needed. They can all be replaced with js native `BigInt`. E.g. in the case `fee: 200n`.

Reply:
- Removed bn.js instances
- Replaced `BigInt` with native js syntax
Added the mentioned changes.